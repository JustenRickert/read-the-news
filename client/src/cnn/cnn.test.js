const {
  __impl: { parseAuthorInformation, parsePublicationDateInformation },
} = require('./')

describe('parsing utils', () => {
  it('parses publication dates', () => {
    const tests = ['Updated 1:13 AM ET, Sat November 2, 2019']
    expect(tests.map(parsePublicationDateInformation)).toMatchSnapshot()
  })

  it('parses authors info', () => {
    const tests = [['Opinion by Lori Harrison-Kahan']]
    expect(tests.map(parseAuthorInformation)).toMatchSnapshot()
  })
})
