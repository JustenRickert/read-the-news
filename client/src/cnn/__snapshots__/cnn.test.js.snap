// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parsing utils parses authors info 1`] = `
Array [
  Array [
    Object {
      "href": null,
      "name": "Lori Harrison-Kahan",
    },
  ],
]
`;

exports[`parsing utils parses publication dates 1`] = `
Array [
  "Sat Nov 02 2019 00:00:00 GMT-0500 (Central Daylight Time)",
]
`;
